docs/advanced.md

```diff
@@ -162,3 +162,61 @@ client = httpx.Client(proxies=proxy)
 # This request will be tunneled instead of forwarded.
 client.get("http://example.com")
 ```
+
+
+## Timeout fine-tuning
+HTTPX library offers many request timeout management options. HTTPX distinguish three types of timeouts: **connect** timeouts, 
+**write** timeouts and **read** timeouts.
+
+* Connect timeout specify time period within which a connection between a client and a server must be established.
+* Write timeout specify the maximum duration before timing out writes of the response.
+* Read timeout specify the maximum duration for reading the entire request.
```

Comment:
And here, it turns out that the read timeout applies to receiving individual *chunks* of the *response*:

```suggestion
* Read timeout specifies the maximum duration to wait for a chunk of data to be received (for example, a chunk of the response body). If HTTPX is unable to receive data within this time frame, a `ReadTimeout` exception is raised.
```